pkg/cmd/experimental/syncgroups/syncgroups.go

```diff
@@ -376,7 +332,7 @@ func (o *SyncGroupsOptions) Run(cmd *cobra.Command, f *clientcmd.Factory) error
 		return err
```

Comment:
How is this print here going to conflict with the print during the sync job?

How do we ensure that someone trying to pipe the output of this sync command to another doesn't have to deal with other garbage?